backend/src/main/java/com/praksa/auction/service/BidService.java

```diff
@@ -1,24 +1,59 @@
 package com.praksa.auction.service;
 
+import com.praksa.auction.dto.BiddingInfoDto;
 import com.praksa.auction.model.Bid;
 import com.praksa.auction.repository.BidRepository;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
+import java.time.DateTimeException;
+import java.util.Date;
+
 @Service
 public class BidService {
```

Comment:
Read more about java doc [here](https://www.baeldung.com/javadoc), every class suppose to have it, also every method in service should have it.